.github/workflows/build.yml

```diff
@@ -128,24 +102,18 @@ jobs:
 
     steps:
       # full checkout for semantic-release
-      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
+      - name: Checkout repository
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
         with:
           fetch-depth: 0
```

Comment:
I think this can also be removed?

Reply:
semantic-release needs a full checkout to build the changelog, there is no way to know how many commits ago was the last tag

Reply:
🙈 sure